### ISSUE TITLE
no domain_blacklist attribute

### ISSUE THREAD

**Issue by zcatbear** [eSentire/cymon-python#3]
I have been trying to use this library to update get a domain_blacklist returning and always get an error.

```
In [4]: api.domain_blacklist('malware')
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-27aa5f593144> in <module>()
----> 1 api.domain_blacklist('malware')

AttributeError: 'Cymon' object has no attribute 'domain_blacklist'
```

I have put in my API key already and I don't see the domain_blacklist option when i do help(api).
I know it is part of the package but I can't seem to see it for some reason. I have installed this through pip and have tried to upgrade it and reinstall it through pip.

**Comment by zcatbear** [comment]
Update, this is because the pypi.python.org version is dated back to 2015. I had to install from source for everything to work correctly.

**Comment by spam-and-abuse** [comment]
So let me rephrase this: could you update package in PyPI? There are number of methods introduced in this client (and in Cymon API) which are not present in PyPI package.
